src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -59,15 +64,14 @@ import {
   timestampToDisplayString,
 } from '../../Utilities/time';
 
-type ImageTableProps = {
-  selectedBlueprint?: string | undefined;
-};
-
-const ImagesTable = ({ selectedBlueprint }: ImageTableProps) => {
+const ImagesTable = () => {
   const [page, setPage] = useState(1);
   const [perPage, setPerPage] = useState(10);
+  const selectedBlueprint = useAppSelector(selectSelectedBlueprint);
+  const blueprintsSearchQuery = useAppSelector(selectBlueprintSearchInput);
```

Comment:
Would you mind changing this to `blueprintSearchInput`? I find it easier to understand the code when the name of the variable in the React component is the same as it is in the store.